tests/Analyzers/CodeStyleAnalyzerFormatterTests.cs

```diff
@@ -6,13 +6,19 @@
 using Microsoft.CodeAnalysis.Tools.Formatters;
 using Microsoft.CodeAnalysis.Tools.Tests.Formatters;
 using Xunit;
+using Xunit.Abstractions;
 
 namespace Microsoft.CodeAnalysis.Tools.Tests.Analyzers
 {
     public class CodeStyleAnalyzerFormatterTests : CSharpFormatterTests
     {
         private protected override ICodeFormatter Formatter => AnalyzerFormatter.CodeStyleFormatter;
 
+        public CodeStyleAnalyzerFormatterTests(ITestOutputHelper output)
+        {
+            TestOutputHelper = output;
```

Comment:
On failure of any tests in the class, the formatter log will be written to the test output.